plugins/chat/spec/jobs/scheduled/auto_join_users_spec.rb

```diff
@@ -13,7 +23,25 @@
 
     job.execute({})
 
+    # should exclude bot / inactive / staged / suspended users
+    expect(Chat::UserChatChannelMembership.where(chat_channel: channel).count).to eq(
+      User.real.not_suspended.not_staged.where(active: true).count,
+    )
```

Comment:
I'm personally not a fan of this test because we can introduce a bug which results in the same count but the test will still pass. I think we should be able to do something like this? 

```suggestion
    expect(Chat::UserChatChannelMembership.where(chat_channel: channel).map(&:user_id)).to contain_exactly(...)
```